### PR TITLE
Documentation restructure and UI guide

### PR DESCRIPTION
**Pull request by avglassman**
With Robin, I've reorganized the documentation, and we've identified some pages that need to be written.

New here:
* Structure: Documentation now has top-bar sections. The more technical (installation, config, etc) is in the Administration guide, and less technical info (UI maneuvers, for now) are in the User guide. The pages have been reorganized/regrouped and lightly edited for clarity.
* The User guide: It's new and quite simple, definitely room for improvement in the future.

I created several pages for Robin and myself to write/co-write when there's time, and once those are ready, this can be the future structure: 
[Proposed documentation nav tree.md](https://github.com/TeskaLabs/seacat-auth/files/13888903/Proposed.documentation.nav.tree.md)

I suspect we may reorganize a bit yet again (mainly to more effectively group information about credentials, tenants, resources, roles etc that is currently present in multiple categories), but for now, I believe that this will be easier to work with. 🙂

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -46,3 +53,58 @@ extra:
...
+  - User guide (UI):
+      - user/index.md
+      - user/key-terms.md
+      - user/login.md
+      - user/credentials.md
+      - user/tenants.md
+      - user/resources.md      
+      - user/roles.md
+      - user/clients.md
+      - user/sessions.md
```
Comment (byewokko) [inline mkdocs.yml:110]: in my view, the User guide is meant for common users who use seacat auth only for logging in and possibly updating their own account settings (email address, passwords etc.). they don't need to know anything about credentials, resources or tenants - all of that is admin stuff.

i think we should only have two pages here - one about login and the other about account settings.

the rest should probably go to the configuration section (for now at least)

File: .DS_Store
Comment (byewokko) [inline .DS_Store:1]: these .DS_Store files shouldn't be part of the merge request :ghost:

**Review by byewokko — CHANGES_REQUESTED** [review]
thanks, this looks great! i have some remarks about the User guide, please see the relevant comment

**Review by ateska — APPROVED** [review]
Let's merge this to see that live

**Comment by byewokko** [comment]
an idea: we could group the basic admin UI pages such as managing tenants, roles and resources under a new section called "Access control". i 'd say access control is all that most admins want from seacat auth.

**Comment by avglassman** [comment]
Updated - Now there is an **Administration guide** section and an **Account help** section (new documents added there, as we discussed). Aleš already approved, @byewokko can we merge and see?

**Review by byewokko — APPROVED** [review]
nice, let's merge it!

**Pull request merged**